activerecord/test/cases/errors_test.rb

```diff
@@ -13,4 +15,19 @@ def test_can_be_instantiated_with_no_args
       end
     end
   end
+
+  def test_association_errors_copied_to_foreign_key
```

Comment:
That's certainly not the right place to put such test. You can move it to `validations_test.rb`.